omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,96 @@
+   :help:`OMERO.figure Help guide <figure.html#info>` (OMERO.figure supports a
+   complete figure creation workflow, including exporting figures into image
+   processing applications for final adjustments - see the
+   :help:`OMERO.figure Help guide <figure.html>` for full details).
+#. Having all the data belong to one user simplifies the UI experience for
+   public users. If necessary, ownership of data can be transferred using the
+   'Chown' privilege (see :doc:`restricted-admins` and
+   :doc:`/users/cli/chown`).
+
+Configuring OMERO.web
+---------------------
+
+#. Configure the OMERO.web filter on the public user by setting the public URL
+   filter to allow 'webclient' (see :doc:`public` and
+   :property:`omero.web.public.url_filter`) so that the full webclient is
+   visible for the public user, and thus also the Data tree with Projects and
+   Datasets is browsable as well as Tags tab and full viewer of the images
+#. In order to have both a public webclient and a closed webclient side by
+   side without affecting the existing closed webclient (and thus giving your
+   existing non-public OMERO users an unchanged user experience in their
+   workflows), a dedicated,
+   :doc:`separate web server <unix/install-web/web-deployment>` for servicing
+   the public workflows can be added and configured to point at an existing
+   OMERO.server.
+
+Data configuration
+------------------
+
+#. Once the data is in the dedicated public group, it can be reorganized and
+   renamed to reflect the publication e.g. Projects can be renamed according
+   to the corresponding figure panels in the manuscript while the names of the
+   Datasets could be retained corresponding to different treatment conditions
+   represented in each figure panel. For example, Project
+   `Schleicher_etal_figure7_c <https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27920>`_
+   contains images underlying the publication Figure panel 7c. Some Projects
+   underlie two publication Figure panels, such as Project
+   `Schleicher_etal_figure2_a_c <https://omero.lifesci.dundee.ac.uk/webclient/?show=project-27917>`_
+   where representative images are shown in the first panel and the
+   corresponding quantification is shown in the next panel. This makes clear
+   which original images are underlying which figure panels in the
+   publication.
+#. Data can also be tagged with OMERO tags to enhance the browsing
+   possibilities through these data for any user with basic knowledge of
+   OMERO. For example, see 
+   `<https://omero.lifesci.dundee.ac.uk/webclient/?show=tag-364188>`_). The
+   tags are highlighting the images displayed in the publication figures as
+   images (the other, non-tagged images in the group are the ones used for
+   analysis which produced the published numerical data).
+
+Configuring URLs
+----------------
+
+#. Create a URL pointing to the first Project (corresponding to the first
```

Comment:
To create a URL, you want to select the Project, then click in right-hand pane onto the chain icon and copy the url from there. Do we have in Help a description of that short workflow ?